admin-dev/themes/new-theme/js/pages/product/edit/product-type-manager.ts

```diff
@@ -32,25 +32,31 @@ import ProductMap from '@pages/product/product-map';
  * is confirmed the form is submitted right away to validate the change and update the page.
  */
 export default class ProductTypeManager {
+  $typeSelector: JQuery;
+
+  $productForm: JQuery;
+
+  productId: number;
+
+  initialType: string | number | string[] | undefined;
```

Comment:
```suggestion
  initialType: string;
```

Type is always a string, and is always defined

Reply:
If TS is not happy about the return of `val()` you can use `standard` as a default fallback, but it shouldn't be necessary.